### PR TITLE
Further user ban implementation

### PR DESCRIPTION
**Pull request by Zaczero**
#33

### CODE REVIEW

**Review by muratcanakcay — APPROVED** [review]
code cleanup ftw!

File: backend/CbtBackend/Services/UserService.cs [hunk 1]
```diff
@@ -64,20 +76,20 @@ public class UserService : IUserService {
             existingUser.Email = userRequest.Email;
         }
 
-        if (userRequest.Banned is bool banned) {
```
[the remark this answers is not in the document]
Comment (Zaczero) [reply backend/CbtBackend/Services/UserService.cs:67]: I believe that creating a new variable for just one assignment is pointless. It's easier to refactor code with the recent change, and it's also more readable to a human being. Also, both solutions are equal in terms of "security" (as if we have cared about it).

[the remark this answers is not in the document]
Comment (Zaczero) [reply backend/CbtBackend/Services/UserService.cs:67]: `.Value` is guaranteed to be non-nullable. It's the same solution but without an extra variable. Just keeping things simple.

**Review by wolftender — APPROVED** [review]
good code

**Pull request merged**